CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Run a `containerdebug` process in the background of each Airflow container to collect debugging information ([#557]).
 
+### Changed
+
+- Bump `stackable-operator` from `0.82.0` to `0.83.0` ([#565]).
```

Comment:
We normally don't document op-rs bumps in here - we only do so when there are user-facing changes

Reply:
https://github.com/stackabletech/airflow-operator/pull/565/commits/f4c49985b03e61c672ab2f59b7a8c8aa7281a810